onadata/libs/tests/test_throttle.py

```diff
@@ -1,20 +1,41 @@
 from django.core.cache import cache
+from django.contrib.auth.models import AnonymousUser, User
 from django.test import TestCase, override_settings
 
 from rest_framework.test import APIRequestFactory
 
-from onadata.libs.throttle import RequestHeaderThrottle
+from onadata.libs.throttle import RequestHeaderThrottle, CustomScopedRateThrottle
+
+class CustomScopedRateThrottleTest(TestCase):
+    def setUp(self):
+        # Reset the cache so that no throttles will be active
+        cache.clear()
+        self.factory = APIRequestFactory()
+        self.throttle = CustomScopedRateThrottle()
+
+    def test_anonymous_users_get_throttled_based_on_uri_path(self):
```

Comment:
@FrankApiyo One trick to make test names shorter is to make use of a docstring e.g

```
def test_anonymous_users(self):
    """Anonymous users  get throttled base on URI path"""
```

Reply:
Thanks